### ISSUE TITLE
Olympus OMD E-M1 images look too dark and colours are off  with standard profile

### ISSUE THREAD

**Issue by da-phil** [Beep6581/RawTherapee#3288]
Hi!

After pulling the latest commits from the repo I noticed that my Olympus OMD E-M1 images look completely different with my standard processing profile.
I'm using an Adobe *.dcp colour profile which made me able to reproduce the in-camera JPGs perfectly.
Now I notice that the images look way too dark (I changed the black point to correct this) and the colours are also a little bit off.
It must be a change in the code in the last 1-2 months, that's the time when I didn't pull new commits and re-build RT. Or maybe a bug?
I'll provide example images later!

Cheers
Phil

**Comment by ClaesGitHub** [comment]
Just a long shot...
- See Tab 3 Color | Color Management, Input Profile. 
Which option button is highlighted: No profile, Camera standard, or Auto-matched camera profile? 
- Under Tab 3 Color | Color Management, Input Profile | DCP: is Base table ticked? 
- Anything else ticked under DCP? 
- Is Tone curve tickable? If so, tick it.

**Comment by da-phil** [comment]
I checked "auto-matched camera profile" and RT successfully detetcs my default Adobe DCP profile (vivid) which the camera also uses for JPG conversion.
I used to check all four boxes:
- tone curve
- look table
- baseline exposure
- base table (greyed out and checked by default)

Now I only get close to the out-of-camera (OOC) JPG picture when I uncheck "baseline exposure".
Here is an example, in RT nothing else than the "Color Management" category settings are set:

OOC picture:
![p5040002_ooc](https://cloud.githubusercontent.com/assets/810285/15293626/22c66b38-1b89-11e6-81be-5dfedaf03f1f.JPG)

all four boxes are checked (way too dark):
![p5040002_all_checked](https://cloud.githubusercontent.com/assets/810285/15293641/3b410146-1b89-11e6-94d3-5a402794c709.jpg)

all four boxes except the "baseline exposure" are checked (this is very close to the OOC picture above):
![p5040002_all_checked_except_baseline_exposure](https://cloud.githubusercontent.com/assets/810285/15293662/545f69ec-1b89-11e6-9ef0-28c571eac57f.jpg)

as I said before, I used to check all  four boxes before to get exactly the OOC result. that's why I'm worried I might have missed an important change / update how to deal with colour profiles or even discovered a bug or malfunctioning of the colour management settings.

**Comment by ClaesGitHub** [comment]
I believe we have different opinions about what _Auto-matched camera profile_ means and does, you and I.
Ref 
https://discuss.pixls.us/t/can-i-match-rt-to-out-of-camera-jpeg/767

http://50.87.144.65/~rt/w/index.php?title=The_Image_Editor_Tab%23Eek.21_My_Raw_Photo_Looks_Different_than_the_Camera_JPEG#Eek.21_My_Raw_Photo_Looks_Different_than_the_Camera_JPEG

http://50.87.144.65/~rt/w/index.php?title=Color_Management#Input_Profile

**Comment by da-phil** [comment]
I don't think we have different opinions on the option "Auto-matched camera profile". I simply copied the Adobe DCP colour profiles from the DNG converter into the install folder of RT, so that RT is always able to select the right colour profile for each camera I use. and this is also covered by the links you posted.

**Comment by Beep6581** [comment]
Which version did they look "correct" in?
Which version are you using now?
Where are the sample raw files and DCPs?
Have you used the same DCP 2 months ago as now? This is to exclude the possibility that Adobe changed their E-M1 DCP file(s).
Why is there a sharp exposure change in the middle image just below the rock in the middle of the stream? 2/3rds from the top.

**Comment by da-phil** [comment]
@Beep6581 sorry for not replying for a long time :/

Actually I don't remember the version where activating "baseline exposure" led to very similar OOC jpg results. I have to look into this again when I've got more time.

Yes, I used the same DCP back then, I'm using it since quite a while now.
I'm using the vivid colour scheme for OOC jpgs in the camera, hence also the vivid DCP in RT.

I don't even notice the sharp exposure change in the middle image. There should be none because I didn't turn on any RT tool which modifies the image.

Here I provided the files in case you want to play around:
https://heliotropium.it/ownclown/index.php/s/hqmiKNQjdu9PZrY
